src/geophires_x/Outputs.py

```diff
@@ -1005,9 +1005,9 @@ def PrintOutputs(self, model: Model):
                 reservoir_parameters.append(OutputTableItem('Reservoir impedance', '{0:10.2f}'.format(model.wellbores.impedance.value / 1000),
                                     model.wellbores.impedance.CurrentUnits.value))
             else:
-                reservoir_parameters.append(OutputTableItem('Reservoir hydrostatic pressure',
-                                                            '{0:10.2f}'.format(model.wellbores.Phydrostaticcalc.value),
-                                                            model.wellbores.Phydrostaticcalc.CurrentUnits.value))
+                reservoir_parameters.append(OutputTableItem('Average reservoir pressure',
+                                                            '{0:10.2f}'.format(np.average(model.wellbores.production_reservoir_pressure.value)),
```

Comment:
1. Per your question offline about backwards compatibility - Does `Average reservoir pressure` need to completely replace `Reservoir hydrostatic pressure`? 
2. In general, calculating properties should be avoided in Outputs as this digs us deeper into the hole of entangled model and view as we've discussed previously. That is - `Average reservoir pressure` should probably be defined as an output parameter and calculated in `Calculate`

Reply:
1) Yes, Average Reservoir Pressure replaces Reservoir Hydrostatic Pressure, because Pressure is no longer a single value (it is an array of pressure over time), and is no longer hydrostatic (could be overpressured).
2) I was following the model that was already there in Outputs.py, but I will change it, as it is a best practice to do as you say here - do the calculations in the Calculate method.  So I changed it - but only for this case. We will clean up the other cases as we come across them. Maybe it would be a good "first programming task" to go through the Outputs.py object and move all calculations made there to the appropriate Calculate method. There is a lot of (minor) calculations made in Output.py.